plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/meta/CubridMetaModel.java

```diff
@@ -53,100 +65,362 @@ public String getTableOrViewName(GenericTableBase base)
+    }
+
+    @NotNull
+    @Override
+    public JDBCStatement prepareSynonymsLoadStatement(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container)
+            throws SQLException {
+        String sql = "select * from db_synonym where synonym_owner_name = ?";
+        final JDBCPreparedStatement dbStat = session.prepareStatement(sql);
+        dbStat.setString(1, container.getName());
+        return dbStat;
+    }
+
+    @Nullable
+    @Override
+    public GenericSynonym createSynonymImpl(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container,
+            @NotNull JDBCResultSet dbResult)
+            throws DBException {
+        String name = JDBCUtils.safeGetStringTrimmed(dbResult, "synonym_name");
+        String description = JDBCUtils.safeGetString(dbResult, "comment");
+        return new CubridSynonym(container, name, description, dbResult);
+    }
+
+    @NotNull
+    @Override
+    public boolean supportsDatabaseTriggers(@NotNull GenericDataSource dataSource) {
+        return true;
+    }
+
+    @NotNull
+    @Override
+    public boolean supportsTriggers(@NotNull GenericDataSource dataSource) {
+        return true;
+    }
+
+    @NotNull
+    @Override
+    public JDBCStatement prepareTableTriggersLoadStatement(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container,
+            @Nullable GenericTableBase table)
+            throws SQLException {
+        String sql = "select t1.*, t2.*, owner.name from db_trigger t1 join db_trig t2 \n"
+                + "on t1.name = t2.trigger_name where owner.name = ? \n"
+                + (table != null ? "and target_class_name = ?" : "");
+        final JDBCPreparedStatement dbStat = session.prepareStatement(sql);
+        dbStat.setString(1, container.getName());
+        if (table != null) {
+            dbStat.setString(2, table.getName());
+        }
+        return dbStat;
+    }
+
+    @Nullable
+    @Override
+    public CubridTrigger createTableTriggerImpl(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container,
+            @Nullable GenericTableBase table,
+            @Nullable String triggerName,
+            @NotNull JDBCResultSet dbResult)
+            throws DBException {
+        String name = JDBCUtils.safeGetString(dbResult, "name");
+        String description = JDBCUtils.safeGetString(dbResult, "comment");
+        return new CubridTrigger(table, name, description, dbResult);
+    }
+
+    @NotNull
+    @Override
+    public JDBCStatement prepareContainerTriggersLoadStatement(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container)
+            throws SQLException {
+        String sql = "select t1.*, t2.*, owner.name from db_trigger t1 join db_trig t2 \n"

```

Comment:
> SQL Error: Semantic: SELECT is not authorized on db_trigger

I can't find this db_trigger table. 
Do you have any documentation links?

Reply:
**db_trigger** table can only be query when login as dba (Database Administration).
Here is the documentation links: https://www.cubrid.org/manual/en/11.2/sql/trigger.html